src/errors.rs

```diff
@@ -72,11 +72,11 @@ pub enum InternalError {
 #[derive(Clone, Eq, PartialEq, Error, Debug)]
 #[allow(missing_docs)]
 pub enum CallerError {
-    #[error("Recieved a message with the wrong recipient ID")]
+    #[error("Received a message with the wrong recipient ID")]
     WrongMessageRecipient,
-    #[error("Recieved a message with the wrong session ID")]
+    #[error("Received a message with the wrong session ID")]
     WrongSessionId,
-    #[error("Recieved a message with the wrong protocol type for this participant (malicious behavior suspected)")]
+    #[error("Received a message with the wrong protocol type for this participant (malicious behavior suspected)")]
```

Comment:
Thank you!! You may be the only person on this project who can spell this word consistently :facepalm: